### PR TITLE
New relic improvements

### PR DESCRIPTION
**Pull request by bhavikkumar**

### CODE REVIEW

**Comment by bhavikkumar** [comment]
I also fixed the issue regarding the metrics bean throwing errors when running from an IDE.

**Comment by bhavikkumar** [comment]
The newrelic agent now comes from the CDN, this only tags a couple of seconds to retrieve which is quicker than cloning a 10MB repository.

File: .travis.yml [hunk 1]
```diff
@@ -5,9 +5,6 @@ jdk:
 - openjdk11
 env:
   global:
-  # Following encrypted environment variables must be present for successful deployment
-  # DEV_ACCOUNT_ID,DEV_ACCESS_KEY_ID,DEV_SECRET_ACCESS_KEY
-  # PROD_ACCOUNT_ID,PROD_ACCESS_KEY_ID,PROD_SECRET_ACCESS_KEY
```
Comment (pushkar-g) [inline .travis.yml]: Maybe we should keep this comment? Or document it somewhere.

**Review by alphafoobar — APPROVED** [review]

**Review by alphafoobar — APPROVED** [review]

File: .travis.yml (hunk 1 above)
Comment (bhavikkumar) [reply .travis.yml]: Reverted this change

**Pull request merged**